docs/dom-testing-library/api-events.md

```diff
@@ -57,6 +57,16 @@ fireEvent.change(getByLabelText(/picture/i), {
     files: [new File(['(⌐□_□)'], 'chucknorris.png', { type: 'image/png' })],
   },
 })
+
+// note: When firing change events please be careful to use valid values.
+// For example if changing a target's value to a date you must abide by ISO 8601
```

Comment:
This isn't 100% correct. It sounds like e.g. you're changing a value from a number to a date but the DOM doesn't make that inference. It will only check this if the input is already a date input. So I would change the wording here to make it clear that we have a date input (not just any input) and that we want to change its value.